### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by glebme**
## Changes
- Update testing steps and context.

## Change reason
This is to update the readme on the current state of testing and what it involves.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,44 +29,48 @@ With the repo cloned, proceed to checkout to the dev branch, `git checkout dev`.
...
+To validate structs, we are utilising a feature of the echo web framework that allows us to couple a validator package to validate structs that contain user inputs from requests. The package is golang's [package validator](https://pkg.go.dev/gopkg.in/go-playground/validator.v9?tab=doc#pkg-index). Everytime validation needs to occur for inserting into a database please use `echo-context.Validator(&struct)` to validate and handle any errors accordingly.
+
+### Unit and Integration Testing
+These tests are written in golang's standard testing package and are written in the same package as the file that they are testing. To have these test run, please ensure docker is running and your containers for development are running. Then to run the actual test, go to the directory that contains the go files and run `go test`. This will run all test in child directories.
```
Comment (sergiomercado19) [inline README.md:66]: You should include how the test files should be named with the *_test.go suffix. For a given program main.go, the test file must be named main_test.go

Comment (glebme) [reply README.md]: Did a brief one liner and directed them to the website for further info.

File: README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ docker-compose down
 
 which kills your containers but keeps your iamges. Be sure to use `docker-compose --help` for any additional help or other options.
 
-If the code has been changed, you need not rebuild using `docker-compose up -d --build` for backend or mongo but as of this moment and vue file changes will need to run the command.
+If the code has been changed, rebuild using `docker-compose up -d --build` if you have made changes to the frontend files. For backend or mongo containers they are reloaded live and any changed made on your system will be automatically refelected in the container and recompiled accoridngly.
```
Comment (sergiomercado19) [inline README.md:44]: This is much better 😄 .... except for the spelling: changed => change, refelected => reflected and accoridngly => accordingly.

File: README.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ The project uses Github Actions for continuous integration and automated testing
...
 
 ### Unit and Integration Testing
-These tests are written in golang's standard testing package and are written in the same package as the file that they are testing. To have these test run, please ensure docker is running and your containers for development are running. Then to run the actual test, go to the directory that contains the go files and run `go test`. This will run all test in child directories. 
+These tests are written in golang's standard testing package and are written in the same package as the file that they are testing. The tests are named `*_test.go` and the testing package has to be imported. For more information please read the Golang documentation for the [package testing](https://golang.org/pkg/testing/). To have these test run, please ensure docker is running and your containers for development are running. Then to run the actual test, go to the directory that contains the go files and run `go test`. This will run all test in child directories.
```
Comment (sergiomercado19) [inline README.md:66]: Awesome!

File: README.md (hunk 2 above)
Comment (glebme) [reply README.md:44]: English is not my strongest subject. Will fix now

**Pull request merged**